src/components/MyRequests/MyRequests.tsx

```diff
@@ -1,7 +1,19 @@
-import { IColumn, SelectionMode, ShimmeredDetailsList } from "@fluentui/react";
-import { Button, makeStyles, tokens } from "@fluentui/react-components";
+import {
+  Button,
+  createTableColumn,
+  DataGrid,
+  DataGridBody,
+  DataGridCell,
+  DataGridHeader,
+  DataGridHeaderCell,
+  DataGridRow,
+  makeStyles,
+  TableCellLayout,
+  TableColumnDefinition,
+  tokens,
+} from "@fluentui/react-components";
 import { AddIcon } from "@fluentui/react-icons-mdl2";
-import { useMyRequests } from "api/RequestApi";
+import { useMyRequests, IInRequest } from "api/RequestApi";
```

Comment:
Great input.  I almost put a comment above the section it was used, and then decided not to.  The next PR I am working will actually be adding a separate Out-processing section, and this will be labeled In-processing.  Wanted to get the PR over to you before I took time figuring out how to best stylize it.  I sent two mock ups to the Product Owner, and they opted for 2 separate section, rather than 1 with an identifier of In or Out processing.  So thanks for pushing through, as that works perfectly :)